bootstrap/kubeadm/api/v1alpha4/kubeadm_types_test.go

```diff
@@ -69,9 +70,9 @@ func TestUnmarshalJSON(t *testing.T) {
 			err := json.Unmarshal([]byte(rt.input), newbts)
 			if rt.expectedError {
 				g.Expect(err).To(HaveOccurred())
-			} else {
-				g.Expect(err).NotTo(HaveOccurred())
+				return
```

Comment:
In case of errors json-iterator might return a struct with some data in, so we are skipping the Equal at the end of the test.
Same in a couple of other tests calling Unmarshal directly